src/app/components/named-entity-relation/named-entity-relation.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, Input } from '@angular/core';
+import { Observable } from 'rxjs';
+import { map } from 'rxjs/operators';
+import { NamedEntity, Relation } from '../../models/evt-models';
+import { register } from '../../services/component-register.service';
+import { NamedEntitiesParserService } from '../../services/xml-parsers/named-entities-parser.service';
+
+@register
+@Component({
+  selector: 'evt-named-entity-relation',
+  templateUrl: './named-entity-relation.component.html',
+  styleUrls: ['./named-entity-relation.component.scss'],
+})
+export class NamedEntityRelationComponent {
+  @Input() data: Relation;
+  @Input() inEntity: boolean;
+
+  // tslint:disable-next-line: variable-name
```

Comment:
Shouldn't be needed

Reply:
If I remove the comment I get the error `"variable name must be in lowerCamelCase, PascalCase or UPPER_CASE (variable-name)"` since the private property name is `_selectedEntity`. Shall I change the name of the variable?

Reply:
Do you need the underscore?